packages/markdown/remark/src/index.ts

```diff
@@ -160,9 +160,8 @@ function prefixError(err: any, prefix: string) {
 	const wrappedError = new Error(`${prefix}${err ? `: ${err}` : ''}`);
 	try {
 		wrappedError.stack = err.stack;
-		// @ts-expect-error
 		wrappedError.cause = err;
```

Comment:
I added `target: ES2022` in the `tsconfig.json` so this line wouldn't need a `// @ts-expect-error`.